src/main/java/barcus/task/Deadline.java

```diff
@@ -0,0 +1,57 @@
+package barcus.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Task with a date to do by
+ */
+public class Deadline extends Task {
+    private static final DateTimeFormatter fromFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm");
```

Comment:
Maybe Deadline and Event can extend of a TimedTask? so that there is no need for multiple formatters.